package.json

```
@@ -4,10 +4,14 @@
   "private": true,
   "scripts": {
     "serve": "./scripts/build_entry.sh -d",
+    "oxy-serve": "./scripts/build_entry.sh -d",
```

It looks like the `oxy` variants don't change anything (with the exception of the deploy script), so we should remove them to avoid confusion.

---

I think that this is actually a typo and it should be different I think the line should be
```suggestion
    "oxy-serve": "./scripts/oxy_build_entry.sh -d",
```